docs/UserGuide.md

```diff
@@ -96,15 +96,25 @@ Examples:
 
 ### Listing all persons : `list`
 
-Shows a list of all persons in the address book, automatically sorted by their priority from HIGH to LOW.
+Shows a list of persons in the address book, automatically sorted by their priority from HIGH to LOW.
```

Comment:
One thing to note is that executing list after sorting by another parameter actually maintains the sorting order, since list shows the last updated list. Perhaps this behaviour could be explained here so the user is not misled to believe that listing always sorts in priority order?

Reply:
Will change it to this:
`...sorted according to the latest sorting order specified by the sort command. If no sorting order has been specified, the list will be sorted by priority from HIGH to LOW.`